.github/workflows/todo_app_ci_pipeline.yml

```diff
@@ -23,13 +23,15 @@ jobs:
         env:
           mongo_db_connection: "${{secrets.MONGO_DB_CONNECTION}}"
           mongo_db_name: "${{secrets.MONGO_DB_NAME}}"
-        run: docker run -e mongo_db_connection -e mongo_db_name my-test-image todo_app/e2e_tests
+          git_client_id: "${{secrets.GIT_CLIENT_ID}}"
+          git_client_secret: "${{secrets.GIT_CLIENT_SECRET}}"
```

Comment:
Presumably you could pass fake values here?

Reply:
yes, forgot that in running tests, auth is disabled